src/api/routes/v1/submission.routes.ts

```diff
@@ -1,14 +0,0 @@
-import * as express from 'express';
-import * as submissionsController from '../../controllers/submission.controller';
-import multer from '../../middlewares/multer';
-import { authenticateJWT } from '../../middlewares/auth';
-const submissionRouter = express.Router();
-
-submissionRouter
-  .route('/upload')
```

Comment:
Why did we have this route initially?

Reply:
This was the automatic grader (i.e upload student submission and automatically grade them)

Reply:
Thanks.